packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -62,15 +62,21 @@ export const createDefaultChatHandlers = memoizeOne(
       onUpdateMessage: async (
         messageId: string,
         content: string,
+        /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
```

Comment:
if options are under cc flag here, it shouldn't be available on line 71 for stable build

Reply:
not sure why stable build still pass...

Reply:
Has `attachment-download` been added to the stabilized features list already and so this line goes through anyway?

Reply:
`not sure why stable build still pass...`
in stable, `options` is undefined, and therefore at line 71 `const updatedMetadata` would be set to `{}`


`Has attachment-download been added to the stabilized features list already and so this line goes through anyway?`
no it hasn't, jim will have a PR to decouple this later